internal/database/migrations/migrations.go

```diff
@@ -81,6 +80,8 @@ func Seed(db *gorm.DB, locationFinder location.LocationFinder) error {
 		}
 		stories[i].Latitude = coordinates.Latitude
 		stories[i].Longitude = coordinates.Longitude
+		// Make sure all stories are public for initially seeded stories
+		stories[i].IsPublic = true
```

Comment:
Dont think we would need to this if we set the default value as false in the gorm model (check my other comment)